network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -416,10 +421,103 @@ class Meta:
         verbose_name = "General Product Page"
 
 
-class BuyersGuidePage(FoundationMetadataPageMixin, Page):
+class BuyersGuidePage(RoutablePageMixin, FoundationMetadataPageMixin, Page):
+    """
+    Note: We'll likely be converting the "about" pages to Wagtail Pages.
+    When that happens, we should remove the RoutablePageMixin and @routes
+    """
+
     template = 'buyersguide/home.html'
     subpage_types = [SoftwareProductPage, GeneralProductPage]
 
+    @route(r'^about/$', name='how-to-use-view')
```

Comment:
do we want to use `r'^about/?$'` just to be sure the trailing `/`, or missing trailing `/`, causes a mishit?

Reply:
Django (and Wagtail by default) enforces the slash at the end whenever possible. Going to `/about` will redirect to `/about/`. But if you have strong feelings about this we can remove the optional slash at the end.

Reply:
🤔 I re-read your comment. I dont like the idea of it purposely causing a mishit due to a missing slash.

Reply:
I think if Django adds the `/` automatically then this should be fine and we can safely land this.